src/sgame/sg_spawn.cpp

```diff
@@ -1105,6 +1106,16 @@ static void SP_worldspawn()
 	G_SpawnStringIntoCVar( "BPBudgetPerMiner", g_buildPointBudgetPerMiner );
 	G_SpawnStringIntoCVar( "BPRecoveryRateHalfLife", g_buildPointRecoveryRateHalfLife );
 
+	G_SpawnString( "luaScript", "", &s );
```

Comment:
I think it's premature to add support for Lua embedded in BSPs... people seem to have an expectation that backward compatibility will be maintained forever for every map. It would suck if we immediately get locked in to any API design flaws